htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/element/TextInput.java

```diff
@@ -37,6 +37,15 @@ public void sendKeys(CharSequence... keys) {
     }
 
     /**
+     * Sets this text input's value.
+     *
+     * @param newValue New value for this text input.
+     */
+    public void setText(String newValue) {
+        getWrappedElement().sendKeys(getClearCharSequence() + newValue);
```

Comment:
I think instead the `getClearCharSequence() +` should be added to `TextInput.sendKeys` method. This way everybody who wants to change input value has no need to prepend it.